lib/src/configuration.dart

```diff
@@ -90,6 +90,9 @@ class Configuration {
   /// downloaded. Please make sure to use the matching version.
   late String? binPath;
 
+  /// Place to download sentry-cli
```

Comment:
We could add the default value in the comment, same as for the other properties.

Reply:
done.